examples/fishjam-chat/screens/PreviewScreen/PreviewScreen.tsx

```diff
@@ -162,7 +162,7 @@ const styles = StyleSheet.create({
   cameraPreview: {
     flex: 6,
     margin: 24,
-    alignSelf: 'stretch',
+    aspectRatio: 9 / 16,
```

Comment:
Could you somehow link this change with FCE-1181? (either by adding todo comment in line above or by adding comment in linear)